### PR TITLE
Auto 440 create job from client test

### PR DESCRIPTION
**Pull request by ShannyWorkiz**

### CODE REVIEW

File: cypress/tests/core1/clients.spec.ts [hunk 1]
```diff
@@ -13,11 +13,22 @@ describe('Clients tests', () => {
...
-});
\ No newline at end of file
+
+    it('When creating new job from client it appears in the jobs report',() =>{
+        let allClientsPage = pageRouter.goToClientsPage();
+        let client = allClientsPage.createClient();
+        let job = client.CreateJobToNewJobPag();
+        cy.get(job.alias).then((jobId) =>{
+        let jobsPage = pageRouter.goToJobsPage();
+        jobsPage.sortClientTableById(false);
```
Comment (IdoAdler2) [inline cypress/tests/core1/clients.spec.ts:30]: @ShannyWorkiz change name

File: cypress/page-objects/Job/createJobPage.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { JobPage } from "./jobPage";
+
+export class CreateJobPage{
+
+    private _jobType: string = '';
+
+    get jobType(){
+        return this._jobType;
```
Comment (IdoAdler2) [inline cypress/page-objects/Job/createJobPage.ts:8]: @ShannyWorkiz No need to getter if not necessary.

File: cypress/page-objects/Job/createJobPage.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { JobPage } from "./jobPage";
+
+export class CreateJobPage{
+
+    private _jobType: string = '';
```
Comment (IdoAdler2) [inline cypress/page-objects/Job/createJobPage.ts:5]: @ShannyWorkiz No need to get on creation.

File: cypress/tests/core1/clients.spec.ts [hunk 4]
```diff
@@ -13,11 +13,22 @@ describe('Clients tests', () => {
...
         pageRouter.goToClientsPage().IsClientsTableContainsClientId(clientId);
         });
     });
-});
\ No newline at end of file
+
+    it('When creating new job from client it appears in the jobs report',() =>{
+        let allClientsPage = pageRouter.goToClientsPage();
+        let client = allClientsPage.createClient();
+        let job = client.CreateJobToNewJobPag();
```
Comment (IdoAdler2) [inline cypress/tests/core1/clients.spec.ts:27]: @ShannyWorkiz U forgot e in the end of the name (pag => page)
CreateJobToNewJobPag(); => CreateJobToNewJobPage();

File: cypress/tests/core1/clients.spec.ts (hunk 1 above)
Comment (ShannyWorkiz) [reply cypress/tests/core1/clients.spec.ts:30]: Fixed

File: cypress/tests/core1/clients.spec.ts (hunk 4 above)
Comment (ShannyWorkiz) [reply cypress/tests/core1/clients.spec.ts:27]: Fixed

File: cypress/page-objects/Job/createJobPage.ts (hunk 3 above)
Comment (ShannyWorkiz) [reply cypress/page-objects/Job/createJobPage.ts:5]: Deleted

**Review by IdoAdler2 — APPROVED** [review]

**Pull request merged**